pydm_launcher/main.py

```diff
@@ -14,6 +14,13 @@ def main():
         nargs='?',
         default=None
         )
+    parser.add_argument(
```

Comment:
Are we going to support more than one stylesheet file to be sent?
If not, let's make this a `--stylesheet` argument and not positional, it makes the launcher and scripts from users more clear and explicit...